Makefile

```
@@ -1,55 +1,86 @@
 	@echo "Available targets:"
-	@awk 'BEGIN {FS = ":.*?## "; OFS=" "} /^[a-zA-Z_-]+:.*?## / {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "  \033[36m%-30s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
 
 # ==============================================================================
 # Environment Setup
 # ==============================================================================
 
 .PHONY: setup
-setup: ## Install dependencies and create .env file from .env.example
-	@echo "🐍 Installing python dependencies with uv..."
+setup: ## Initialize project: install dependencies, create .env file and pull required Docker images
```

checkmake (0.2.2)

[warning] Target body for "setup" exceeds allowed length of 5 (22).

(maxbodylength)